Dockerfile

```diff
@@ -10,6 +10,11 @@ RUN wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/d
 RUN curl -sSL https://install.python-poetry.org | python3 -
 ENV PATH="/root/.local/bin:$PATH"
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt bullseye-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
```

Comment:
move line 13,14 just under line 7 and just add postgresql-client-14 to list of packages that are needed

Reply:
`gcc` and `make` is not needed `build-essential` is adding those